src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/Article.tsx

```diff
@@ -1,18 +1,27 @@
-import { Box, Image } from "@artsy/palette"
+import { Box, Image, Text } from "@artsy/palette"
 import { DiscoveryArticle } from "Apps/ArtAdvisor/07-Curated-Discovery/types"
+import { RouterLink } from "System/Components/RouterLink"
+import { resize } from "Utils/resizer"
 import { FC } from "react"
 
+const IMAGE_HEIGHT = 200
+
 interface ArticleRailsProps {
   article: DiscoveryArticle
 }
 
 export const Article: FC<ArticleRailsProps> = ({ article }) => {
+  const resizedImage = resize(
+    article.imageUrl || "https://www.artsy.net/images/missing_image.png",
+    { height: IMAGE_HEIGHT }
+  )
+
   return (
-    <Box>
-      <Image src={article.imageUrl} height={300} />
-      <a href={`${article.href}`} target="_blank">
-        {article.title}
-      </a>
-    </Box>
+    <RouterLink to={article.href} textDecoration="none" overflow="hidden">
```

Comment:
Maybe doesn't matter for a prototype, but this will reroute the current window instead of opening a new window, correct? This felt sub-optimal and was what led me to falling back to the vanilla anchor tag as I didn't see an obvious way to replicate that behavior with out router link.